boot-qemu.sh

```diff
@@ -242,7 +242,10 @@ function setup_qemu_args() {
 # Invoke QEMU
 function invoke_qemu() {
     rm -rf "${ROOTFS}"
-    zstd -d "${ROOTFS}".zst -o "${ROOTFS}"
+    zstd -q -d "${ROOTFS}".zst -o "${ROOTFS}"
+
+    printf "\n\033[01;32m%s\033[0m %s\n\n" "QEMU location:" "$(dirname "$(command -v "${QEMU[*]}")")"
+    printf "\033[01;32m%s\033[0m %s\n\n" "QEMU version:" "$("${QEMU[@]}" --version | head -n1)"
```

Comment:
can you use or define functions like `green` and `red` above? L6-12

Reply:
Done, PTAL.